<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GMA</class>
 <widget class="QWidget" name="GMA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GMA</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="GMA_genPseudoCode_btn">
     <property name="text">
      <string>Gen. Pseudo Code</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>211</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Change Gamma LUT (instant)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="GMA_changeGLUT_lb"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTabWidget" name="GMA_curveTabs_tw"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
